spec/factories/submitted_enrollments_factory.rb

```diff
@@ -9,7 +9,14 @@
     next if ot.to_sym == :unknown
 
     factory :"submitted_#{ot}", parent: :confirmed_random_pending do
+      reference_number { FloodRiskEngine::ReferenceNumber.create }
+      submitted_at { Time.current }
+
       after(:create) do |object|
+        object.enrollment_exemptions.each do |ee|
```

Comment:
Just checked this with my WIP branch (feature/enrollment-actions) and it changes the default status of the enrollment_exemption. Should be okay to change my code, just wondering what this change does?

Reply:
This is the bit: `ee.status = 1`

Reply:
Ahh, it wasn't intended to change the status – more replicate the behaviour of the old `enrollment.submit` function. I'll see if I can tweak it.

Reply:
Just checked and turns out we didn't need that change 🤷‍♀️ Maybe I changed a test that was relying on it later or something!